presets/index.js

```diff
@@ -7,9 +7,9 @@ module.exports = (options = {}) => {
     alchemy: [`wss://eth-mainnet.ws.alchemyapi.io/v2/${options.alchemyId}`, `https://eth-mainnet.alchemyapi.io/v2/${options.alchemyId}`],
     infuraGoerli: [`wss://goerli.infura.io/ws/v3/${options.infuraId}`, `https://goerli.infura.io/v3/${options.infuraId}`],
```

Comment:
Do we need to remove the Goerli one too?

Reply:
looks like mainnet and all of the mainnet testnets still support ws: https://docs.infura.io/infura/networks/ethereum/how-to/choose-a-network